### DISCUSSION TITLE
Is there any plan to support pyarrow?

### DISCUSSION THREAD

**Discussion by goodwanghan** [pyscript/pyscript#1592] [Q&A] [answered]
Hi,

I understand pyarrow is not supported because it doesn't have a pure python wheel. But this is such an important package used by many other packages. Is there any plan to support pyarrow in pyscript?

Thanks
Han

**Comment by sskagemo** [comment, marked as answer]
If I understand it correctly, PyScript will automatically support pyarrow if pyodide supports it, and there is a separate issue for pyodide discussing pyarrow: https://github.com/pyodide/pyodide/issues/2933
